later/lib/presentation/cubit/home_cubit/home_cubit.dart

```diff
@@ -0,0 +1,23 @@
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:injectable/injectable.dart';
+
+import 'package:later/domain/usecases/watch_all_records_use_case.dart';
+import 'home_state.dart';
+
+@lazySingleton
+class HomePageCubit extends Cubit<HomeState> {
+  HomePageCubit({required this.useCase}) : super(HomeEmpty());
+
+  final WatchAllRecordsUseCase useCase;
+
+  void fetchRecords() {
+    emit(HomeLoading());
+    var streamRecords = useCase.call();
```

Comment:
If you define the `call` method in a class, you are able to do `useCase()`, not `useCase.call()`. This is the only method like this